### ISSUE TITLE
BUG: Cannot redefine property: 1 & disappearing of fields

### ISSUE THREAD

**Issue by baur** [grafana/grafana#24272]
**What happened**:
**How to reproduce it (as minimally and precisely as possible)**:

[play](https://s6.gifyu.com/images/sombug.gif?target=_blank) gif

![image](https://user-images.githubusercontent.com/2898361/81036016-3eb26d80-8ebf-11ea-84a8-b2aa985354e7.png)

**Environment**:
- Grafana version: 7.0 beta1
- Data source type & version: MSSQL
- OS Grafana is installed on: Windows
- User OS & Browser: Chrome

**Comment by torkelo** [comment]
What are your bar gauge options? (under display).And can you show what your raw query results are (in the table)

**Comment by baur** [comment]
What are your bar gauge options? 

- by default 

no matter horizontal or vertical

![image](https://user-images.githubusercontent.com/2898361/81039884-f2215f00-8ecb-11ea-96b7-6533dc692ac6.png)

And can you show what your raw query results are?

![image](https://user-images.githubusercontent.com/2898361/81039828-d6b65400-8ecb-11ea-8d39-7887777155be.png)

```
{
  "request": {
    "url": "api/tsdb/query",
    "method": "POST",
    "data": {
      "from": "1588638180430",
      "to": "1588659780431",
      "queries": [
        {
          "refId": "A",
          "intervalMs": 60000,
          "maxDataPoints": 1140,
          "datasourceId": 2,
          "rawSql": "select * from v where valuetypeid=1 order by typeid,[paramName],valuetypeid",
          "format": "table"
        }
      ]
    }
  },
  "response": {
    "results": {
      "A": {
        "refId": "A",
        "meta": {
          "rowCount": 37,
          "sql": "select * from v where valuetypeid=1 order by typeid,[paramName],valuetypeid"
        },
        "series": null,
        "tables": [
          {
            "columns": [
              {
                "text": "paramName"
              },
              {
                "text": "typeid"
              },
              {
                "text": "name"
              },
              {
                "text": "valuetypeid"
              },
              {
                "text": "1"
              },
              {
                "text": "2"
              },
              {
                "text": "3"
              }
            ],
            "rows": [
              [
                0,
                1,
                "Количество запуска",
                1,
                5557,
                5588,
                6520
              ],
 ...
```

**Comment by torkelo** [comment]
What fields are disappearing? Looks correct given the data

**Comment by baur** [comment]
What fields are disappearing? Looks correct given the data

fields are disappearing in UI -> Table -> Transform -> Filter
Please, [play ](https://s6.gifyu.com/images/sombug.gif?target=_blank) gif file ...

**Comment by baur** [comment]
Please see also #24231, I'm commented ...

**Comment by baur** [comment]
got new error bug message 

![image](https://user-images.githubusercontent.com/2898361/81041103-c3f14e80-8ece-11ea-8a45-8d5ab1b2b401.png)

but nothing was is changed, just **table panel** change to **bar gauge**

**Comment by baur** [comment]
Got new error message:
**Cannot read property 'subRows' of null** with table panel
all of this errors related with **Transform->Filter by name**

**Comment by torkelo** [comment]
Can you replicate this in 7.0.1 ?

**Comment by baur** [comment]
Can not replicate due to bug #25122 
Transform table not works at all, neither with existing nor with new

**Comment by torkelo** [comment]
ok, the bug you found was fixed in 7.0.1 (before we broke the transformations tab), try nighly master build (it has working transform tab)
